float64[] a
float64[] b
int32 n_reps
string exp_name
---
bool success
string message